pkg/action/validate.go

```diff
@@ -21,12 +21,25 @@ import (
 
 	"github.com/pkg/errors"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/cli-runtime/pkg/resource"
 
 	"helm.sh/helm/v3/pkg/kube"
 )
 
-func existingResourceConflict(resources kube.ResourceList) error {
+var accessor = meta.NewAccessor()
+
+const (
+	appManagedByLabel              = "app.kubernetes.io/managed-by"
+	appManagedByHelm               = "Helm"
```

Comment:
```yaml
apiVersion: v1
data:
  kubernetesServiceHost: pre.master.thanos.sogou
  kubernetesServicePort: "6443"
kind: ConfigMap
metadata:
  annotations:
    meta.helm.sh/release-name: configmap-test
    meta.helm.sh/release-namespace: bizdev
  creationTimestamp: "2020-05-07T11:33:46Z"
  labels:
    123.a: 456.b
    app.kubernetes.io/managed-by: Helm
  name: configmap-test
  namespace: bizdev
  resourceVersion: "359608781"
  selfLink: /api/v1/namespaces/bizdev/configmaps/configmap-test
  uid: 6109837e-d753-41fb-be76-ad3145341978
```

I think `app.kubernetes.io/managed-by` value in lower camel case will be better, like `app.kubernetes.io/managed-by: helm`, to make the style consistency.